Sources/ConsoleKitCommands/Async/AsyncCommandGroup.swift

```diff
@@ -60,8 +62,8 @@ extension AsyncCommandGroup {
             context.console.print(self.help)
         }
 
-        let padding = self.commands.map { $0.key }.longestCount + 2
-        if self.commands.count > 0 {
+        let padding = (self.commands.max(by: { $0.key.count < $1.key.count })?.key.count ?? 0) + 2
```

Comment:
```suggestion
        let padding = (self.commands.map(\.key.count).max() ?? 0) + 2
```

Reply:
BTW this is a suggestion for performance, not a nit. It just happens to be cleaner too.